tests/components/philips_js/test_device_trigger.py

```diff
@@ -80,6 +78,6 @@ async def test_if_fires_on_turn_on_request(
     )
 
     await hass.async_block_till_done()
-    assert len(calls) == 1
-    assert calls[0].data["some"] == mock_device.id
-    assert calls[0].data["id"] == 0
+    assert len(service_calls) == 2
```

Comment:
I dont like this. What is this extra call here? What is to stop generic HA changes from suddenly changing this to a 3?

Reply:
Ps. It needs to look up the service call to "test automation". If not found then error out. Not sure this ends up better than what there was before.

Reply:
The change from 1 to 2 here is due to `media_player.turn_on` being logged in addition to `test.automation`.
It is called explicitely in the test on line 73... so I think it is sufficiently clear.

Reply:
I've adjusted it to make it explicit what is in the first call, and what is in the second.